R/draws.R

```diff
@@ -68,6 +68,10 @@
 #' Note that LOCF is generally not a principled imputation method and should only be used when appropriate 
 #' for the specific covariate.
 #'
+#' Please note that there is no special provisioning for "baseline". If you do not want baseline
+#' observations to be included in the model then these should be removed in advance from `data`. 
```

Comment:
```suggestion
#' observations to be included in the model then these should be removed in advance from the outcome variable in `data`. 
```

Reply:
@gowerc I am not 100% sure we need to mention this, but I am fine with it. 

Just "remove them from `data`" might be confusing since one might want to include the baseline outcome as covariate in the model, so in this case the user should not remove it from `data`.